### PR TITLE
Cleanup turbulence + add ref to dycoms ex

### PR DESCRIPTION
**Pull request by asraero**
1) Add rising bubble and density current to gpu tests 
2) Add ref for dycoms example @kpamnany 
3) Remove old dycoms from test-set @smarras79 
4) Field info for turbulence.jl + vreman fix for gpu
5) Update vtk output in drivers to use dg.auxstate @simonbyrne

### CODE REVIEW

**Comment by asraero** [comment]
bors try

File: src/Atmos/Model/turbulence.jl [hunk 1]
```diff
@@ -165,12 +179,12 @@ end
...
   DT = eltype(state)
   ∇u = ∇transform.u
-  αijαij = sum(abs2, ∇u)
+  αijαij = sum(∇u .^ 2)
   @inbounds normS = strain_rate_magnitude(S)
   f_b² = squared_buoyancy_correction(normS, ∇transform, aux)
   βij = f_b² * (aux.turbulence.Δ)^2 * (∇u' * ∇u)
   @inbounds Bβ = βij[1,1]*βij[2,2] - βij[1,2]^2 + βij[1,1]*βij[3,3] - βij[1,3]^2 + βij[2,2]*βij[3,3] - βij[2,3]^2 
-  return state.ρ * max(0,m.C_smag^2 * 2.5 * sqrt(abs(Bβ/(αijαij+eps(DT))))) 
+  return state.ρ * (m.C_smag^2 * 2.5) * sqrt(abs(Bβ/(αijαij+eps(DT))))
```
Comment (lcw) [inline src/Atmos/Model/turbulence.jl]:
```suggestion
  return state.ρ * (m.C_smag^2 * DT(2.5)) * sqrt(abs(Bβ/(αijαij+eps(DT))))
```

**Comment by asraero** [comment]
bors try

**Pull request merged**